src/plugins/orangehrmOpenidAuthenticationPlugin/config/routes.yaml

```diff
@@ -0,0 +1,31 @@
+apiv2_auth_providers:
+  path: /api/v2/auth/providers
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ POST, GET, DELETE ]
+  defaults:
+    _api: OrangeHRM\OpenidAuthentication\Api\ProviderAPI
+
+apiv2_auth_provider:
+  path: /api/v2/auth/providers/{id}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, PUT ]
+  defaults:
+    _api: OrangeHRM\OpenidAuthentication\Api\ProviderAPI
+
+##########################
+#        Screens
+##########################
+add_auth_provider:
+  path: admin/addAuthProvider
+  controller: OrangeHRM\OpenidAuthentication\Controller\SaveSocialMediaAuthProviderController::handle
+  methods: [ GET ]
+
+edit_auth_provider:
+  path: admin/editAuthProvider
+  controller: OrangeHRM\OpenidAuthentication\Controller\EditSocialMediaAuthProviderController::handle
+  methods: [ GET ]
+
+auth_oidc_login:
+  path: /auth/oidcLogin
```

Comment:
openidauth/openIdCredentials was the previous used one. but since we don't have that dynamic redirect url I added this.  This can change in next PR since not using for these APIs